### PR TITLE
addition of abstracts and canonical types for FCU

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -36,6 +36,26 @@ SD:
   - supply_air_damper_percentage_command
   implements:
   - CONTROL
+  
+EPM:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:40]: Not used elsewhere. Remove

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -36,6 +36,26 @@ SD:
...
+  uses:
+  - power_status
+  - voltage_sensor
+  - current_sensor
+  - power_sensor
+  - energy_accumulator
+  implements:
+  - MONITORING
+
+RDM:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:52]: Not used elsewhere. Remove.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -353,6 +373,7 @@ VSFC:
   uses:
   - speed_frequency_command
   - speed_frequency_sensor
+  - speed_percentage_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:376]: Remove, cannot be used here.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 4]
```diff
@@ -4611,4 +4611,17 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+
+#AHU canonical#  
+AHU_DFSS_DFVSC_FDPM_CHWDC_SRC_DFRMM:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4617]: Youve defined this without important AHU characteristics, particularly outside air control. Is this an AHU or a FCU?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 5]
```diff
@@ -29,6 +29,7 @@ FCU_DFSS_CSP_CHWDC:
   - CSP
   - CHWDC
 
+
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:32]: If this is just whitespace diff between PRs, please revert.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 5 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:32]: This is just a white space....

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 4 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4617]: It is a ceiling suspended AHU and do not contain any outside air control associated to IT. This is particularly serving the IDF rooms.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 2 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:52]: EPM abstracts removed from the current pull request...

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 3 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:376]: the field has been deleted in the VSFC abstract.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 4 above)
Comment (tasodorff) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4617]: then it is a FCU. please adjust.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 6]
```diff
@@ -1451,4 +1452,47 @@ FCU_RTC_RHDHS_DTM_DFSS:
...
+FCU_CHWDC_DFSS_DXSC_DXZTC_EVFSM_REFPM_ZHC_DFRMM_CHWDT:
+  description: "CRAC with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - CHWDC
+  - DFSS
+  - DXSC
+  - DXZTC
+  - EVFSM
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1467]: explain this. evaporative fan separate from the discharge fan? the evaporator fan in a typical crac unit is the discharge fan. how is this separate?

Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1467]: EFVSM has been replaced by existing DFVSC and the associated telemetries have been deleted.

**Comment by tasodorff** [comment]
please fix merge errors @RathishKJ

**Review by tasodorff — APPROVED** [review]

**Pull request merged**